### ISSUE TITLE
Java layer: the configuration is wrong

### ISSUE THREAD

**Issue by zhexuany** [syl20bnr/spacemacs#3293]
I trie but spacemacs return symbool error. 
Initially, the configuration for eclim is 
(setq eclim-eclipse-dirs '("~/opt/eclipse")
( eclim-executable "~/opt/eclipse/eclim"))
But this one does not work on my computer. I modified it in the form like
(setq eclim-eclipse-dirs "~/opt/eclipse"
eclim-executable "~/opt/eclipse/eclim")
but with different path. 
And now it work.
Additionally, if someone installed eclipse by home-brew, then the default path will be very different.

**Comment by zhexuany** [comment]
@syl20bnr  I think we should change this a little bit, what do you think?

**Comment by robbyoconnor** [comment]
This may not be the case for EVERYBODY -- this is a case of you not configuring things properly.  Those are examples -- it is assumed people know to replace  strings appropriately.  

This isn't a bug at all. People are required to actually think and realize "hmm this isn't right, let me change it" -- aka know how your system is configured.

**Comment by robbyoconnor** [comment]
This may not be the case for EVERYBODY -- this is a case of you not configuring things properly.  Those are examples -- it is assumed people know to replace  strings appropriately.  

This isn't a bug at all. People are required to actually think and realize "hmm this isn't right, let me change it" -- aka know how your system is configured. 

I don't even think this needs to stay open -- sorry for bluntness. The docs say, emphasize was added by me: 
> Then set the =Eclipse= and =Eclim= paths in =dotspacemacs/user-config=,
**for instance**:

>#+BEGIN_SRC elisp
> (setq eclim-eclipse-dirs '("~/opt/eclipse")
>                         '(eclim-executable "~/opt/eclipse/eclim"))
> #+END_SRC

**Comment by robbyoconnor** [comment]
The fact of the matter is this: People aren't stupid (at least I hope not) -- they know how to find the path to eclipse and to place it where it's supposed to go. I don't view this as a bug -- sorry you can't copy/paste and go, but this isn't a bug.

**Comment by TheBB** [comment]
@robbyoconnor *Read* the issue before you respond at least.

```lisp
(setq eclim-eclipse-dirs '("~/opt/eclipse")
                         '(eclim-executable "~/opt/eclipse/eclim"))
```

This was never a valid `setq` form. @zhexuany has corrected it to

```lisp
(setq eclim-eclipse-dirs "~/opt/eclipse"
      eclim-executable "~/opt/eclipse/eclim")
```

which is right. He didn't even change the paths!

> People are required to actually think and realize "hmm this isn't right, let me change it"

Yeah, which is fine for values, but our docs don't have to be intro to programming 101 "figure out what's wrong" pop-quizzes.

**Comment by robbyoconnor** [comment]
Ah :) Nevermind -- I removed the comment :)

**Comment by StreakyCobra** [comment]
Should be corrected by #3311

**Comment by zhexuany** [comment]
Thanks you guys. I gonna closed this issue.
